app/controllers/cart.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  actions: {
+    //empty the local storage and then force a refresh of the cart template
+    emptyCart() {
+      localStorage.removeItem('cart');
+      this.get('target.router').refresh();
```

Comment:
instead of doing this, what do you think of just overwriting the `model` object on this controller which should automatically update your template (this will obv change if we add a cart service)